guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -1,47 +1,62 @@
 [id="importing-kickstart-repositories_{context}"]
-= Importing Kickstart Repositories
+= Importing Kickstart Repositories for Red{nbsp}Hat Enterprise Linux{nbsp}{rhel-version}
 
-Kickstart repositories are not provided by the Content ISO image. To use Kickstart repositories in your disconnected {Project}, you must download a binary DVD ISO file for the version of Red{nbsp}Hat Enterprise Linux that you want to use and copy the Kickstart files to {Project}.
+To import kickstart repositories for Red{nbsp}Hat Enterprise Linux{nbsp}{rhel-version}, complete the following steps on {Project}.
```

Comment:
Capitalization not consistent with other instances. Kickstart?

Reply:
Done